### PR TITLE
JSON to YAML instructions

### PR DESCRIPTION
**Pull request by Kentoseth**
how to convert json to yaml

### CODE REVIEW

**Comment by TingPing** [comment]
All usage of this repo implies cloning it, so I'd rephrase it a bit to just include the command to run. (And then its executable so you can remove `python3`)

**Comment by Kentoseth** [comment]
> All usage of this repo implies cloning it, so I'd rephrase it a bit to just include the command to run. (And then its executable so you can remove `python3`)

Let me know if I got the right executable if the repo is cloned.
